### PR TITLE
Git clone step missing 

### PR DESCRIPTION
**Pull request by krishnakalyan3**
@sgugger doing this for the first time. Could you please check and let me know if everything is okay. 

Many Thanks

### CODE REVIEW

**Comment by sgugger** [comment]
Hi @krishnakalyan3 , it seems you forgot to run `nbdev_install_git_hooks` after cloning the repo, so you have committed metadata in the notebook you changed that isn't necessary (and risk merge conflicts). COuld you run that command and adjust your PR?

**Comment by krishnakalyan3** [comment]
sure. Will do!

File: nbs/index.ipynb [hunk 1]
```diff
@@ -98,5 +99,5 @@
   }
  },
  "nbformat": 4,
- "nbformat_minor": 2
+ "nbformat_minor": 4
```
[the remark this answers is not in the document]
Comment (sgugger) [reply nbs/index.ipynb]: No this is part of the essential metadata that needs to be kept, so no worries.

**Comment by sgugger** [comment]
Thanks!

**Comment by krishnakalyan3** [comment]
@sgugger thanks for the review :)

**Pull request merged**